src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/session/SessionPrisonWing.kt

```diff
@@ -13,7 +13,7 @@ import javax.persistence.Table
 data class SessionPrisonWing(
 
   @Column(nullable = false)
-  val prisonId: String,
+  val prisonId: Long,
```

Comment:
will remove this code in separate review and then merge that into this branch for this review!